plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -270,4 +273,37 @@ public void testCreateSchema()
         // verify DROP SCHEMA for non-existing schema
         assertQueryFails("DROP SCHEMA " + schemaName, format("line 1:1: Schema '.*\\.%s' does not exist", schemaName));
     }
+
+    @Test
+    @Override
+    public void testRenameTableUnqualifiedName()
+    {
+        // override needed to provide valid location for schema (it needs to be s3:// based). By default schema is assigned hdfs:// location
```

Comment:
Can we have a method in BCT to fill in required table properties (default: none), so that we don't need to override such nice and verbose test method?

Reply:
Thank you for seeing this cleaner way to customize the tests for delta lake.